### DISCUSSION TITLE
Error Message:  Error searching messages, API responded with status 403! 	{"message":"Missing Access","code":50001} CoreException	{}

### DISCUSSION THREAD

**Discussion by willrrs** [victornpb/undiscord#593] [1. Questions & Answers]
When trying to use index.json file, to mass delete all the messages i get the following result:
Error searching messages, API responded with status 403!
	{"message":"Missing Access","code":50001}
CoreException	{}

Ive tried this on firefox and on chrome.
whats the reason for this and can i fix this?

**Comment by willrrs** [comment]
After trying to look into the index file itself: i deleted all the null values and it still didnt work.
I then reduced the file to only 2 indices and this still didnt work and it came up with a 400 error this time.
However, i have no saved passwords and its still not working.
Anyone know what to do?

**Reply by Jinglebell2** [reply]
No idea and I also want to do the same

**Reply by GauravASB** [reply]
any solution yet?
